source/includes/steps-upgrade-shared-cluster.yaml

```
@@ -0,0 +1,54 @@
+title: Shutdown All Running Emulators
```

"Shutdown" is a noun. "Shut down" is the verb phrase.
```suggestion
title: Shut Down All Running Emulators
```